lib/battle/BattleInfo.h

```diff
@@ -1,151 +1,154 @@
-/*

```

Comment:
Whole file changed, problem with line endings?

Reply:
I made small test, by using push -f I checked for LF, CRLF, CR, but it's not resolve the problem, any ideas what it can be?

Reply:
set 
``` 
[core]
	safecrlf = warn
```
in .gitconfig and recommit ff3a2148

Reply:
Thanks! Will check tommorow.

Reply:
Well with `safectrlf = warn` each time when I want  commit I get `fatal: bad numeric config value 'warn' for 'core.autocrlf': invalid unit`
Are you sure there must be `warn`?

Reply:
It should work, see https://git-scm.com/docs/git-config#git-config-coresafecrlf

Reply:
Huh, there were 2 problems. 
1) CLion seems to have some bug since it doesn't change line endings.
2)You was close but instead of `git config --global core.autocrlf warn` change to `git config --global core.autocrlf false` and this help me a bit.